apps/system/test/unit/download_notification_test.js

```diff
@@ -96,14 +96,25 @@ suite('system/DownloadNotification >', function() {
     });
 
     test('The download starts', function() {
+      notification.state = 'started';
       assert.isFalse(NotificationScreen.addNotification.called);
       download.state = 'downloading';
       download.listeners[0]();
       assertUpdatedNotification(download);
 
-      sinon.assert.calledWithMatch(NotificationScreen.addNotification, {
-        noNotify: true
-      });
+      assert.isUndefined(
+        NotificationScreen.addNotification.args[0].noNotify);
+    });
+
+    test('The download continues', function() {
+      notification.state = 'downloading';
+      assert.isFalse(NotificationScreen.addNotification.called);
+      download.state = 'downloading';
+      download.listeners[0]();
+      assertUpdatedNotification(download);
+
+      var calls = NotificationScreen.addNotification.getCalls();
+      assert.isTrue(calls[0].args[0].noNotify);
```

Comment:
nit: let's use the sam way to access the call like `NotificationScreen.addNotification.args[0].noNotify`